src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Utf8JsonReader.SingleSegment.Tests.cs

```diff
@@ -0,0 +1,132 @@
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void MultipleShortSingleLineComment_CommentHandling_Allow() => GenericReadTest(s_multipleShortSingleLineComment, JsonCommentHandling.Allow);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void SingleLongSingleLineComment_CommentHandling_Allow() => GenericReadTest(s_singleLongSingleLineComment, JsonCommentHandling.Allow);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void SingleEmptyMultiLineComment_CommentHandling_Skip() => GenericReadTest(s_singleEmptyMultiLineComment, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void MultipleEmptyMultiLineComment_CommentHandling_Skip() => GenericReadTest(s_multipleEmptyMultiLineComment, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void SingleShortMultiLineComment_CommentHandling_Skip() => GenericReadTest(s_singleShortMultiLineComment, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void MultipleShortMultiLineComment_CommentHandling_Skip() => GenericReadTest(s_multipleShortMultiLineComment, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void SingleLongMultiLineComment1_CommentHandling_Skip() => GenericReadTest(s_singleLongMultiLineComment1, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void SingleLongMultiLineComment2_CommentHandling_Skip() => GenericReadTest(s_singleLongMultiLineComment2, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void SingleEmptySingleLineComment_CommentHandling_Skip() => GenericReadTest(s_singleEmptySingleLineComment, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void MultipleEmptySingleLineComment_CommentHandling_Skip() => GenericReadTest(s_multipleEmptySingleLineComment, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void SingleShortSingleLineComment_CommentHandling_Skip() => GenericReadTest(s_singleShortSingleLineComment, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void MultipleShortSingleLineComment_CommentHandling_Skip() => GenericReadTest(s_multipleShortSingleLineComment, JsonCommentHandling.Skip);
+
+        [Benchmark(OperationsPerInvoke = Iterations)]
+        public void SingleLongSingleLineComment_CommentHandling_Skip() => GenericReadTest(s_singleLongSingleLineComment, JsonCommentHandling.Skip);
+
+        private static void GenericReadTest(byte[] dataUtf8, JsonCommentHandling commentHandling)
+        {
+            for (int i = 0; i < Iterations; i++)
```

Comment:
I would remove `Iterations` here and also remove `OperationsPerInvoke`. They are not needed for a test like this.